safer-ffi-gen/src/without_std/error.rs

```diff
@@ -0,0 +1,12 @@
+use alloc::string::{String, ToString};
+
+pub fn set_last_error<E>(_: E)
+where
+    E: ToString,
+{
+}
+
+#[safer_ffi::ffi_export]
+pub fn last_error() -> safer_ffi::String {
+    String::from("Error message is not recorded in no_std").into()
```

Comment:
Make this empty string so that the error message isn't taking up space maybe?

Reply:
I rebased and added a commit to address that. Thank you.